client/src/components/Suggestions/BulkCreateMonumentSuggestions/BulkCreateMonumentSuggestion/BulkCreateMonumentSuggestion.js

```diff
@@ -1,38 +1,52 @@
 import * as React from 'react';
 import './BulkCreateMonumentSuggestion.scss';
+import { connect } from 'react-redux';
 import { Card } from 'react-bootstrap';
 import CreateMonumentSuggestions from '../../CreateMonumentSuggestions/CreateMonumentSuggestions';
 import { Link } from 'react-router-dom';
 import { getUserFullName } from '../../../../utils/string-util';
+import { Role } from '../../../../utils/authentication-util';
 
 /**
  * Presentational component for displaying a BulkCreateMonumentSuggestion
  */
-export default class BulkCreateMonumentSuggestion extends React.Component {
+class BulkCreateMonumentSuggestion extends React.Component {
+
+    static mapStateToProps(state) {
+        return {
+            session: state.session
+        };
+    }
```

Comment:
Do you think the user role should be passed down rather than having this component aware of Redux?

Reply:
These get rendered in a couple different places so I'd have to pass it through a bunch of components that don't actually care about the session at all. In this case, I think it's better just to get the session where we need it (and also less time consuming to write)